packages/menu/src/MenuItem.ts

```diff
@@ -39,6 +39,7 @@ import checkmarkStyles from '@spectrum-web-components/icon/src/spectrum-icon-che
 import type { Menu } from './Menu.js';
 import { MutationController } from '@lit-labs/observers/mutation-controller.js';
 import type { Overlay } from '@spectrum-web-components/overlay';
+import { SlottableRequestEvent } from '@spectrum-web-components/overlay/src/slottable-request-event.js';
```

Comment:
I am not sure whether we have expanded the readme to include this usage but we can definately add some context on this for the consumers in a good document on how and when to use it?

Reply:
This unlocks the last bit of scope for the initial test interaction with the PS team. Once that test cycle is complete, the last piece of confirmation before documentation/"official release" will be confirming how the directives work in partnership with requests for content in slots other than the default slot AND THEN I will start writing documentation with all available speed. One part of that documentation will also be internal outlining path for future use of this form of "enhancement testing process".